Marketplace/src/main/resources/db/migration/V1__initTables.sql

```diff
@@ -0,0 +1,99 @@
+CREATE TABLE categories (
```

Comment:
You need to format you sql for better readability. You can use some online formatter as starting point, e.g. https://www.dpriver.com/pp/sqlformat.htm